client/src/components/ElementEditor/Element.js

```diff
@@ -2,9 +2,15 @@
 
 import React, { Component, PropTypes } from 'react';
 import { elementType } from 'types/elementType';
+import { bindActionCreators, compose } from 'redux';
 import { inject } from 'lib/Injector';
 import i18n from 'i18n';
 import classNames from 'classnames';
+import { connect } from 'react-redux';
+import { change } from 'redux-form';
```

Comment:
Is this (and the redux-form dependancy) actually needed?

Reply:
Without them we cannot update the form state for the active tab. Unless you can come up with a more elegant solution, we do need them. 'connect' allows us to connect the Element component to the Redux store and change allows us to dispatch an action.

Reply:
Yep - I was too tired last night - missed the use of it within `mapDispatchToProps` 👍